src/ArrowEffects.cpp

```diff
@@ -126,7 +126,7 @@ void ArrowEffects::Update()
 
 			int iStartCol = iColNum - iTornadoWidth;
 			int iEndCol = iColNum + iTornadoWidth;
-			iEndCol = Rage::clamp( iStartCol, 0, pStyle->m_iColsPerPlayer-1 );
+			iStartCol = Rage::clamp( iStartCol, 0, pStyle->m_iColsPerPlayer-1 );
```

Comment:
I thought I already fixed this. Oh well: hopefully it doesn't cause a git conflict.